RELEASES.md

```diff
@@ -1,3 +1,148 @@
+Version 1.79.0 (2024-06-13)
+==========================
+
+<!-- FIXME: unsorted T-types relnotes -->
+- [check `FnDef` return type for WF](https://github.com/rust-lang/rust/pull/115538/)
+- [rework opaque type region inference](https://github.com/rust-lang/rust/pull/116891/)
+- [instantiate higher ranked goals outside of candidate selection](https://github.com/rust-lang/rust/pull/119820/)
+- [Pass list of defineable opaque types into canonical queries](https://github.com/rust-lang/rust/pull/122077/)
+- [Eagerly instantiate closure/coroutine-like bounds with placeholders to deal with binders correctly](https://github.com/rust-lang/rust/pull/122267/)
+- [Use fulfillment in method probe, not evaluation](https://github.com/rust-lang/rust/pull/122317/)
+- [Make inductive cycles always ambiguous](https://github.com/rust-lang/rust/pull/122791/)
+- [Enforce closure args + return type are WF](https://github.com/rust-lang/rust/pull/123531/)
+- [Fix trait solver overflow with `non_local_definitions` lint](https://github.com/rust-lang/rust/pull/123594/)
+- [More DefineOpaqueTypes::Yes](https://github.com/rust-lang/rust/pull/123794/)
+
+<a id="1.79.0-Language"></a>
+
+Language
+--------
+- [Stabilize inline `const {}` expressions.](https://github.com/rust-lang/rust/pull/104087/)
+- [Prevent opaque types being instantiated twice with different regions within the same function.](https://github.com/rust-lang/rust/pull/116935/)
+- [Stabilize WebAssembly target features that are in phase 4 and 5.](https://github.com/rust-lang/rust/pull/117457/)
+- [Add the `redundant_lifetimes` lint to detect lifetimes which are semantically redundant.](https://github.com/rust-lang/rust/pull/118391/)
+- [Stabilize the `unnameable_types` lint for public types that can't be named.](https://github.com/rust-lang/rust/pull/120144/)
+- [Enable debuginfo in macros, and stabilize `-C collapse-macro-debuginfo` and `#[collapse_debuginfo]`.](https://github.com/rust-lang/rust/pull/120845/)
+- [Propagate temporary lifetime extension into `if` and `match` expressions.](https://github.com/rust-lang/rust/pull/121346/)
+- [Restrict promotion of `const fn` calls.](https://github.com/rust-lang/rust/pull/121557/)
+- [Split `refining_impl_trait` lint into `_reachable`, `_internal` variants.](https://github.com/rust-lang/rust/pull/121720/)
+- [Stabilize associated type bounds (RFC 2289).](https://github.com/rust-lang/rust/pull/122055/)
+
```

Comment:
```suggestion
- [Check return types of function types for well-formedness](https://github.com/rust-lang/rust/pull/115538)
- [Rework `impl Trait` lifetime inference](https://github.com/rust-lang/rust/pull/116891/)
- [Change inductive trait solver cycles to be ambiguous](https://github.com/rust-lang/rust/pull/122791)

```

Reply:
The first of those at least seems like it should be in compat notes? Not sure though. Maybe both?